test/e2e/storage/testpatterns/testpattern.go

```diff
@@ -333,6 +333,16 @@ var (
 		VolType:        DynamicPV,
 		AllowExpansion: true,
 	}
+
+	// NtfsDynamicPVAllowExpansion is TestPattern for "Dynamic PV (default fs)(allowExpansion)"
+	NtfsDynamicPVAllowExpansion = TestPattern{
+		Name:           "Dynamic PV (ntfs)(allowExpansion)",
+		VolType:        DynamicPV,
+		AllowExpansion: true,
```

Comment:
@gnufied I think this is a future cleanup we should think about. Testing allowexpansion field is mainly testing the admission controller and not the actual driver, so I'm not sure we really need to test the volume expansion disallowed case in the driver testsuite. We should just use the driver test capability to determine if run the test or not.

Reply:
opened tracking issue https://github.com/kubernetes/kubernetes/issues/95888